smailyforprestashop.php

```diff
@@ -348,6 +429,7 @@ public function hookDisplayBackOfficeHeader()
             Media::addJsDef(
                 array(
                     'controller_url' => $this->context->link->getAdminLink($this->controllerAdmin),
+                    'smaiy_rss_url' => Context::getContext()->link->getModuleLink('smailyforprestashop', 'SmailyRssFeed'),
```

Comment:
Typo in the variable name - is `smaiy_rss_url`, but should be `smaily_rss_url`.

Reply:
Somehow also misspelled it in javascript, because of that it worked.
Fixed